web/src/routes/(user)/search/+page.svelte

```diff
@@ -107,6 +110,36 @@
   const handleSelectAll = () => {
     selectedAssets = new Set(searchResultAssets);
   };
+
+  export const loadNextPage = async () => {
+    console.log('loadPage', curPage, term);
```

Comment:
There are three console log left here which I presume need to get removed

Reply:
Oops